### PR TITLE
MRD-2085 - prison api search

### PR DESCRIPTION
**Pull request by robinandrewdevilliers**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/PrisonApiControllerTest.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          .body(
+            BodyInserters.fromValue(prisonOffenderSearchRequest(nomsId)),
+          )
+          .headers {
+            (listOf(it.authToken(roles = listOf("ROLE_MAKE_RECALL_DECISION"))))
+          }
+          .exchange()
+          .expectStatus().isOk,
+      )
+      println(response)
```
Comment (DarrenW-HMPPS) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/PrisonApiControllerTest.kt:37]: Has this been left by mistake?

**Review by DarrenW-HMPPS — CHANGES_REQUESTED** [review]
There are various bits of PPUD Automation API work in here. Are they intentionally mixed in?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/PrisonApiControllerTest.kt (hunk 1 above)
Comment (robinandrewdevilliers) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/PrisonApiControllerTest.kt:37]: yes tks

**Comment by robinandrewdevilliers** [comment]
> There are various bits of PPUD Automation API work in here. Are they intentionally mixed in?

**Comment by robinandrewdevilliers** [comment]
The code coverage issue is due to a timeout method  that can't be covered in integration testing as I would need to reduce the timeout configured in application-test.yaml to 0.  And that would fail the other tests anyway.  The reason its passing in other clients is probably because they represent less than 20% of the lines in the file.

**Pull request merged**